bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/basic/ActionConditions.java

```diff
@@ -77,24 +77,43 @@
     private static @Nullable JsonElement brightnessExists(@Nullable JsonElement value) {
         if (value != null && value.isJsonPrimitive() && value.getAsJsonPrimitive().isNumber()) {
             int intVal = value.getAsInt();
-            if (intVal > 0 && intVal < 99) {
+            if (intVal > 0 && intVal <= 100) {
                 return value;
             }
+            return null;
         } else {
             LOGGER.debug("Could not parse brightness. Value '{}' is not an int", value);
         }
+        return value;
+    }
+
+    /**
+     * Check if the value is a color which can be send to Color channel.
+     * If not returns a null
+     *
+     * @param command
+     *
+     * @param value
+     * @return
+     */
+    private static @Nullable JsonElement isHSV(Command command, @Nullable JsonElement value) {
```

Comment:
The name of this method is inappropriate for what it returns.

Reply:
this is fixed now